### ISSUE TITLE
Add tile info to inspect

### ISSUE THREAD

**Issue by bcamper** [tangrams/tangram-play#582]
The selection info returned with a feature includes information on which tile it came from, in `selection.feature.tile`. It would be useful to have this available in the inspect pop-up for debugging.

**Comment by louh** [comment]
We haven't displayed a nested object before in the inspection popup, so with `feature.tile` being a nested object (with `feature.tile.coords`) it raises the question of whether I display it like a "flattened" object, or put in a more generic ability to render objects of arbitrary nesting depth, or just pick and choose which property or properties are the most useful to display. Since it looks like some of the properties contain the same information (e.g. `feature.tile.key` concatenates `feature.tile.source + feature.tile.style_zoom + feature.tile.coords.x + feature.tile.coords.y + feature.tile.coords.z`) do I need to show all of them? or would only a few of them be useful in the inspection context?

Basically, what is the most relevant information you need from `selection.feature.tile`? And would you have any comments about the most effective way of presenting it?

**Comment by bcamper** [comment]
Thanks, actually, since we already have the data source name, we probably only need the `coords.key`, which is the slash-delimited string of its XYZ components. This is mainly to make it easier to see which tile a feature came from (can only be done with some console debugging right now).

**Comment by louh** [comment]
Just merged, but let me know if you have feedback!

**Comment by bcamper** [comment]
LGTM
